docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -489,128 +724,209 @@ Combinations of environment variables and their uses.
 
 ```sh
 sudo ifconfig lo0 alias 127.0.0.2
 sudo ifconfig lo0 alias 127.0.0.3
 ```
 
-Add two more nodes to the cluster using the `join` option.
+Add two more nodes to the cluster using the `join` option, as follows:
 
 ```sh
-./bin/yugabyted start --advertise_address=127.0.0.2 --join=127.0.0.1 --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node2 --cloud_location=aws.us-east.us-east-2a
-./bin/yugabyted start --advertise_address=127.0.0.3 --join=127.0.0.1 --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node3 --cloud_location=aws.us-east.us-east-3a
+./bin/yugabyted start --secure --advertise_address=127.0.0.2 --join=127.0.0.1 --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node2 --cloud_location=aws.us-east.us-east-2a
+./bin/yugabyted start --secure --advertise_address=127.0.0.3 --join=127.0.0.1 --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node3 --cloud_location=aws.us-east.us-east-3a
 ```
 
 ### Destroy a local multi-node cluster
 
-Destroy the above multi-node cluster.
+To destroy the multi-node cluster, execute the following:
 
 ```sh
-./bin/yugabyted destroy --base_dir=/Users/username/yugabyte-{{< yb-version version="preview" >}}/data1
-./bin/yugabyted destroy --base_dir=/Users/username/yugabyte-{{< yb-version version="preview" >}}/data2
-./bin/yugabyted destroy --base_dir=/Users/username/yugabyte-{{< yb-version version="preview" >}}/data3
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node2
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node3
 ```
 
 ### Create a multi-zone cluster
 
-To create a multi-node cluster, you start the first node by running the `yugabyted start` command, passing in the `--cloud_location` and `--fault_tolerance` flags to set the node location details, as follows:
+To create a multi-node secrue cluster, start the first node by running the `yugabyted start` command, using the `--secure` flag and passing in the `--cloud_location` and `--fault_tolerance` flags to set the node location details, as follows:
 
 ```sh
-./bin/yugabyted start --advertise_address=<host-ip> --cloud_location=aws.us-east.us-east-1a --fault_tolerance=zone
+./bin/yugabyted start --secure --advertise_address=<host-ip> --cloud_location=aws.us-east.us-east-1a --fault_tolerance=zone
 ```
 
-Start the second and the third node on two separate VMs:
+Create certificates for the second and third virtual machine (VM) for SSL and TLS connection, as follows:
 
 ```sh
-./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-east.us-east-2a --fault_tolerance=zone
+./bin/yugabyted cert generate_server_certs --hostnames=<IP_of_VM_2>,<IP_of_VM_3>
+```
+
+Manually copy the generated certificates in the first VM to the second and third VM.
```

Comment:
Manually copy the generated certificates in the first VM to the second and third VM, as follows: